### PR TITLE
fix(ui): add workaround component to fix the hover issue on IconButton

### LINKED ISSUE #149: [Bug] Icon hover not displayed properly using the SMUI IconButton
# Describe the bug
The `IconButton` component from the SMUI library renders improperly on hover. The button's appearance changes unexpectedly, causing visual inconsistencies.

## To Reproduce
Import the `IconButton` component from the SMUI library.
Use the `IconButton` component within a Svelte file.
Hover over the `IconButton`.
Observe the rendering behavior.
Expected behavior
The `IconButton` should maintain a consistent appearance on hover, without any unexpected visual changes.

## Additional context
The issue was observed while using the `IconButton` component in a Svelte project.
The problem may be related to the CSS styles applied to the `IconButton` component.
Ensure that the SMUI library is up-to-date and check for any known issues or updates related to the `IconButton` component.

### PR DESCRIPTION
**Pull request by michelguerin**
This closes #149 

Add a new custom IconButton component as workaround + storybook documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,15 +9,17 @@
...
+		"install:dependencies": "pnpm install:core && pnpm install:ui",
+		"//====== STORYBOOK ======//": "",
+		"storybook": "pnpm install:ui && pnpm --filter @oscd-plugins/ui storybook",
 		"//====== PLUGIN INSTALL ======//": "",
-		"install:type-designer": "pnpm install:core && pnpm install:ui && pnpm --filter @oscd-plugins/type-designer i",
+		"install:type-designer": "pnpm install:dependencies && pnpm --filter @oscd-plugins/type-designer i",
 		"//====== PLUGIN STAND ALONE MODE ======//": "",
-		"dev:type-designer": "pnpm install:type-designer && pnpm --filter @oscd-plugins/type-designer dev",
+		"dev:type-designer": "pnpm --filter @oscd-plugins/type-designer dev",
+		"dev:type-designer:install": "pnpm install:type-designer && pnpm --filter @oscd-plugins/type-designer dev",
```
Comment (salvar3nga) [inline package.json:22]: What's the idea for splitting install & "non install" scripts? How about whenever we run dev it automatically installs the dependecies as well?

File: packages/ui/src/components/icons/icon-wrapper.svelte [hunk 2]
```diff
@@ -1,25 +1,49 @@
...
+// local variables
+const cssFixedStyles = {
+	'svg-padding': ''
+}
 
 //====== REACTIVITY ======//
 
 $: draw = getIcon(icon)
+$: cssFixedStyles['svg-padding'] = setPadding(isCustomIconButton)
+// dynamic inline styles
```
Comment (salvar3nga) [inline packages/ui/src/components/icons/icon-wrapper.svelte:27]: comment can go?

File: package.json (hunk 1 above)
Comment (michelguerin) [reply package.json:22]: My original attempt was to have a flag that installs the dependencies on the first launch in a same terminal session. 
I fire this multiple times a day without having to install everything, everytime.

But I can't make it work (export variables from within the package.json scripts seems to be scoped to its command), that's why two commands (one for instantiation, the other for running the project faster)

File: packages/ui/src/components/icons/icon-wrapper.svelte (hunk 2 above)
Comment (michelguerin) [reply packages/ui/src/components/icons/icon-wrapper.svelte:27]: yep ! It appears in multiple file, as I wanted to emphasis this logic, but finally I think that the names are clear enough.

**Pull request merged**